package.nls.json

```diff
@@ -21,6 +21,7 @@
     "AWS.configuration.description.logLevel": "The AWS Toolkit's log level (changes reflected on restart)",
     "AWS.configuration.description.onDefaultRegionMissing": "Action to take when a Profile's default region is hidden in the Explorer. Possible values:\n* `add` - shows region in the explorer\n* `ignore` - does nothing with the region\n* `prompt` - (default) asks the user what they would like to do.",
     "AWS.configuration.description.s3.maxItemsPerPage": "Controls how many S3 items are listed before showing a node to `Load More...`.\nThis corresponds to the `MaxKeys` requested in a single call to S3. [Learn More](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html#AmazonS3-ListObjectsV2-response-MaxKeys)",
+    "AWS.configuration.description.s3.defaultRegion": "Sets the default region to make requests to AWS S3",
```

Comment:
Be careful about the wording here. This makes it seem like it will apply for all S3 operations, but we haven't made the required changes for this to be true (we can also avoid this by not needing a configurable).